cmd/kubelet/app/options/options.go

```diff
@@ -315,6 +326,8 @@ func (f *KubeletFlags) AddFlags(mainfs *pflag.FlagSet) {
 	f.ContainerRuntimeOptions.AddFlags(fs)
 	f.addOSFlags(fs)
 
+	fs.BoolVar(&f.RegisterNode, "register-node", f.RegisterNode, "Register the node with the apiserver. If --kubeconfig is not provided, this flag is irrelevant, as the Kubelet won't have an apiserver to register with.")
```

Comment:
nit: since we're keeping this as a flag for now, let's put it right back below dynamic-config-dir so that it doesn't show up in the git diff